src/main/java/org/folio/verticles/KafkaConsumersVerticle.java

```diff
@@ -53,32 +56,111 @@ public KafkaConsumersVerticle(KafkaConfig kafkaConfig,
   public void init(Vertx vertx, Context context) {
     super.init(vertx, context);
     context.put("springContext", springContext);
-    ediExportOrdersHistoryKafkaHandler = new EdiExportOrdersHistoryAsyncRecordHandler(context, vertx);
   }
 
   @Override
   public void start(Promise<Void> startPromise) {
-    log.info("Kafka config: {}", kafkaConfig);
+    log.info("start:: starting kafka consumer verticle. kafka config: {}", kafkaConfig);
+
+    createConsumers()
```

Comment:
not sure about usign single verticle for multiple caonsumers.

Each verticle in Vert.x typically runs on its own event loop thread. By deploying each Kafka consumer in a separate verticle, you take full advantage of the multicore architecture of your system, as each consumer can potentially run on a separate thread, achieving true parallel processing of consumed messages.

Can you search please in our github, I think we do not have cases when reusing the same verticle for multiple consumers

Reply:
Looked at other implementation and found that they also used similar approach, one verticle for all consumers. I think vertx run AbstractVerticle that is partent class for all consumer verticle
https://github.com/folio-org/mod-circulation-storage/blob/28aa0b72a3e651c8e5fccfa0ad8f7a222287d27e/src/main/java/org/folio/EventConsumerVerticle.java#L38
https://github.com/folio-org/mod-orders/blob/3eefce4d7ffd27f328bba23f974f1499ca81ee7b/src/main/java/org/folio/verticle/DataImportConsumerVerticle.java#L73
https://github.com/folio-org/mod-source-record-manager/blob/95da2781b71d0423604b866ac7ee0f91bbfe20ac/mod-source-record-manager-server/src/main/java/org/folio/verticle/AbstractConsumersVerticle.java#L48

Reply:
okay got it, thinking what is better have 2 verticles:  Item Create, Item update and (Item delete in future), and Holding Create, Holding Update and (Holding Delete in future) events or use single verticle.
More like approach with 2 verticles(one all holdings events, another for all item events) to have better parallelism I think.
For mod-users we even have 3 separate verticles for the same entity: https://github.com/folio-org/mod-users/tree/master/src/main/java/org/folio/verticle

Reply:
Seems this style is a little bit confusing. Behind the scenes, this structure is using same verticle for all like current approach. Extra Verticles providing specific topic and handler. From my opinion verticles in mod-users are overengineered by having three different verticle for just prodiving different topic and handles. I have doubt it can achieve any performance benefit from this

Regarding your opinion, it make sense to have different verticles for different kafka topic, one for inventory one for export for better clarify.